### ISSUE TITLE
Bad performance on CityScapes dataset.

### ISSUE THREAD

**Issue by makman7** [AlexMa011/pytorch-polygon-rnn#2]
I trained the model on the CityScapes dataset up to 26 epochs but worst prediction while testing (predicting some lines).

**Comment by AlexMa011** [comment]
How about the result on training set? Like, the accuracy? The training dataset of small cropped images contains about 48k images. In my experience, the training loss can be down to nearly zero.

**Comment by makman7** [comment]
Accuracy is 0.05 and loss 5.8 at step 5.44k. 
I checked the performance on different epochs now it has decent results :) but still. Did you compared the repo stats with paper?

**Comment by AlexMa011** [comment]
Glad you get decent results. I did compare the results given the bounding box of the object on 500 validation images. The results are better than 2017 polygon-rnn, but worse than 2018  poygon-rnn++.
